client/stylesheets/shared/_embed.scss

```diff
@@ -123,5 +123,13 @@
 		a.components-button:not(.is-primary) {
 			color: $gray-text;
 		}
-	}
+    }
+}
+
+// for the embed page with the new navigation enabled
+.woocommerce-feature-enabled-navigation.woocommerce-embed-page {
+    #screen-meta,
+    #screen-meta-links {
+        top: $header-height-with-new-nav;
```

Comment:
This should probably be wrapped in a viewport breakpoint:
```
@include breakpoint( '<782px' ) {
}
```
As it's misaligned now when not displaying on a phone:
<img width="1365" alt="Screen Shot 2021-01-22 at 1 55 24 PM" src="https://user-images.githubusercontent.com/2240960/105527035-92691f00-5cb9-11eb-99c7-4df82662b64c.png">


There are also some lint errors:
<img width="463" alt="Screen Shot 2021-01-22 at 1 39 38 PM" src="https://user-images.githubusercontent.com/2240960/105527082-9e54e100-5cb9-11eb-907c-cf1c6d7f6812.png">

Reply:
Thank you! I will address them soon.

Reply:
@louwie17 sorry for the delay. The change is in https://github.com/woocommerce/woocommerce-admin/pull/6153/commits/5951f945c18aeea987a7a01bb0140edbf8aa5044

Reply:
@louwie17 The change is in https://github.com/woocommerce/woocommerce-admin/commit/b4cc15262f66c90c662e9dcc4cf87f90c7ebc054 👍

Reply:
@louwie17 Hi ✋ Could you take a look at the change when you get a chance, please? I'm waiting for this PR to test https://github.com/woocommerce/woocommerce-admin/pull/6118. Thank you 🙏

Reply:
Ooops sorry @moon0326 my bad :/ I will get on that today.